test/fuzz/blockchain/init-corpus/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/golang/protobuf/proto"
+
+	bc "github.com/tendermint/tendermint/blockchain"
+	bcproto "github.com/tendermint/tendermint/proto/tendermint/blockchain"
+	types "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+// Creates "corpus" directory in the current path and writes corpus data.
+
+func main() {
+	// create "corpus" dir if not exists
+	corpusDir := filepath.Join(".", "corpus")
+	if err := os.Mkdir(corpusDir, 0755); err != nil && !os.IsExist(err) {
+		log.Fatalf("can't create %s: %v", corpusDir, err)
+	}
+
+	// write corpus
+	messages := []struct {
+		filename string
+		msg      proto.Message
+	}{
+		{"status-request", &bcproto.StatusRequest{}},
+		{"status-response", &bcproto.StatusResponse{Base: 0, Height: 200}},
+		{"block-request", &bcproto.BlockRequest{Height: 100}},
+		{"block-response", &bcproto.BlockResponse{Block: &types.Block{}}},
+		{"no-block-response", &bcproto.NoBlockResponse{Height: 50}},
+	}
+
+	for _, msg := range messages {
```

Comment:
Sorry, I should probably add this to the readme. The idea with corpus data is to help fuzzier to understand what bytes sequences are semantically valid (e.g. if we're testing PNG decoder, then we would put black-white PNG into corpus directory; same with blockchain reactor - we put blockchain messages into corpus so that fuzzier can use them).